.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,145 @@
+name: CI/CD
+
+on:
+  pull_request: # Runs whenever a pull request is created or updated
+  push: # Runs whenever a commit is pushed to the repository
+    branches: [master, develop, beta, hotfix/*] # ...on any of these branches
+  workflow_dispatch: # Allows you to run this workflow manually from the Actions tab
+
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+env:
+  CXX: g++-4.8
+  FASTLY_ACTIVATE_CHANGES: true
+  FASTLY_PURGE_ALL: true
+  NODE_ENV: production
+  SKIP_CLEANUP: true
+
+jobs:
+  determine-environment:
```

Comment:
Please share if you know of a better way to do branch-specific secrets! This seems silly...

Reply:
I wonder if setting "Environments" in GitHub is a better approach. 
https://github.com/scratchfoundation/scratchjr-website/blob/develop/.github/workflows/deploy.yml

Reply:
Right! I (now) remember seeing that and thinking it might work for www... and then I promptly forgot, I guess. 😅
This code is doing basically the same thing -- setting `environment` -- but it's doing a lot more work to get there. I like your way better. Thanks!